storefront/containers/ProductCard/ProductCard.tsx

```diff
@@ -55,4 +55,5 @@ const ImageContainer = styled.div`
   position: relative;
   width: 100%;
   padding-bottom: 100%; /* forces square aspect ratio */
+  cursor : pointer;
```

Comment:
This is nice, but at the moment I can't click on the card at the moment.

What are you plans when clicking on this card? routing to product details page? If yes you need #6 first.